servatrice/src/servatrice_database_interface.cpp

```diff
@@ -247,7 +247,7 @@ AuthenticationResult Servatrice_DatabaseInterface::checkUserPassword(Server_Prot
         if (!usernameIsValid(user, reasonStr))
             return UsernameInvalid;
 
-        if (checkUserIsBanned(handler->getAddress(), user, reasonStr, banSecondsLeft))
+        if (checkUserIsBanned(handler->getAddress(), user, reasonStr, banSecondsLeft, clientId))
```

Comment:
Can you reorder the parameters to `checkUserIsBanned` so that the out params are at the end?